src/main/java/org/snomed/otf/owltoolkit/domain/Relationship.java

```diff
@@ -148,9 +148,26 @@ public boolean isConcrete() {
 	public static final class ConcreteValue {
 
 		public enum Type {
-			INTEGER,
-			DECIMAL,
-			STRING;
+			INTEGER("INTEGER", "int"),
+			DECIMAL("DECIMAL", "dec"),
+			STRING("STRING", "str");
+
+			private final String name;
```

Comment:
`name` field is not needed. This is already part of every `enum`, see `name()` method. The `toString()` method also returns the name which is taken from the enum literal, e.g. `INTEGER`.

Reply:
Thanks @kaicode  - the name field has been removed.